src/main/kotlin/no/nav/syfo/personstatus/infrastructure/database/repository/PersonOversiktStatusRepository.kt

```diff
@@ -307,19 +307,12 @@ class PersonOversiktStatusRepository(private val database: DatabaseInterface) :
         val initials = searchQuery.initials.value.toList()
         val baseQuery =
             "SELECT * FROM PERSON_OVERSIKT_STATUS p WHERE (p.oppfolgingstilfelle_end + INTERVAL '16 DAY' >= now() OR $AKTIV_OPPGAVE_WHERE_CLAUSE) AND p.fodselsdato = ? AND "
-        val nameQuery =
-            "p.name ILIKE ? AND " + initials.drop(1).joinToString(" AND ") { "p.name ILIKE ?" }
+        val nameQuery = "p.name ILIKE ?"
+        val initialsSearchString = initials.joinToString(separator = "%", postfix = "%")
```

Comment:
Så gitt eksempelet `ABC` vil queryen se slik ut?
```
p.name ILIKE A%B%C%
```
Vil dette kunne potensielt gi flere treff? `Abrac Olsen Hanse` vil vel dukke opp for eksempel? Kan være jeg har misforstått hvordan den blir seende ut

Reply:
Ja, bug/feature. Oppdatert sånn at søkestrengen blir A% B% C% nå. Da vil det funke bedre?

(la til en test for det også)

Reply:
Må kanskje vurdere om vi skal ta høyde for dobbeltnavn med bindestrek. Blir fort mange permutasjoner siden de kan forekomme både som fornavn og etternavn.

Reply:
Hm, sånn sett vil kanskje den gamle måten å gjøre dette på fungere bedre? Da ville vel `Ask-Bernt Charlie` dukket opp, nå sier vi vel at det er nødt å være på formen `A% B% C%` ? 🤔

Reply:
Verken den "gamle" eller nye måten å søke på gir treff på dette tilfellet (dvs ABC der det er bindestrek og ikke space mellom to de fornavnene).